docs/use-cases/azure.md

```diff
@@ -52,23 +52,129 @@ az aks create \
 
 ## Flux Installation with Azure DevOps Repos
 
-Ensure you can login to [dev.azure.com](https://dev.azure.com) for your proper organization, and create a new repo to hold your
-flux install and other necessary config.
-
-There is no bootstrap provider currently for Azure DevOps Repos,
-but you can clone your Azure Repo, then use the [Generic Git Server](../guides/installation.md#generic-git-server)
-guide to manually bootstrap Flux. (It must be a Git repo; TFVC Repos are not supported by source-controller)
-Take note of the Azure DevOps specific section within the guide.
-
-If you use the generated SSH deploy key from `flux create source git`, ensure it is an RSA key (not an elliptic curve).
-Make sure to use the `libgit2` provider for all `GitRepository` objects fetching from Azure Repos since they use Git Protocol v2.
-
-Whether you're using the generated SSH deploy key or a Personal Access Token, the credentials used by
-Flux will need to be owned by an Azure DevOps User with access to the repo.
-Consider creating a machine-user and granting it granular permissions to access what's needed.
-This allows changing user access without affecting Flux.
-Since PAT's expire on Azure DevOps, using a machine-user's login password to authenticate with HTTPS and `libgit2`
-can be a good option that avoids the need to renew the credential while also having the benefit of more granular permissions.
+Ensure you can login to [dev.azure.com](https://dev.azure.com) for your proper organization,
+and create a new repo to hold your Flux install and other necessary config.
```

Comment:
```suggestion
and create a new repository to hold your Flux install and other Kubernetes resources.
```

maybe?

(know this has been adapted from the previous revision, but reading this now I think the "necessary" can create doubt)